### DISCUSSION TITLE
How can we improve findability for pathways/WikiPathways/Communities in Google (or other search engines)?

### DISCUSSION THREAD

**Discussion by fehrhart** [wikipathways/wikipathways-help#59] [Ideas]
Chris and me had some discussions over how to make clinicians and biomedical researchers actually find and use the content we create here in WikiPathways - or other more specialised bioinformatics knowledge bases. So, we just googled some questions around rare disease pathways and also used ChatGPT (see below for a detailed question and its answer - in short, NOT including WP RD community) to get some answers. What was mostly showing up were scientific articles, WP was not shown on first or second pages. 

The main question is - how can we make content of WikiPathways showing up higher in Google or other search engines? 

Can you tell me what to best use if I look for molecular pathways describing rare diseases
ChatGPT
When researching molecular pathways for rare diseases, it's important to use a combination of reliable resources and tools to gather accurate and up-to-date information. Here's a step-by-step guide on what to use:
1.	Online Databases and Resources:
o	OMIM (Online Mendelian Inheritance in Man): A comprehensive resource that provides information on genetic disorders, including molecular pathways, clinical features, and references to relevant literature.
o	Orphanet: A European database dedicated to rare diseases and orphan drugs, which often includes information about molecular pathways associated with rare diseases.
o	PubMed: A database of scientific articles, including a vast amount of research related to rare diseases. Searching for the name of the rare disease along with terms like "molecular pathway" or "pathogenesis" can yield relevant results.
2.	Scientific Journals:
o	Look for recent articles in peer-reviewed journals that focus on the specific rare disease you're interested in. Journals like "Orphanet Journal of Rare Diseases" and "Rare Diseases" may have relevant information.
3.	Bioinformatics Tools:
o	GeneCards: Provides comprehensive information on genes, including their function, associated pathways, and relevant publications.
o	KEGG (Kyoto Encyclopedia of Genes and Genomes): Offers pathway maps and information about molecular interactions and networks.
o	Reactome: A curated knowledgebase of molecular pathways and biological processes.
4.	Genetic and Medical Databases:
o	ClinVar: Contains information about variations in human genes and their clinical significance, which can be helpful in understanding the molecular basis of rare diseases.
o	HGMD (Human Gene Mutation Database): Focuses on disease-causing mutations and their associated diseases.
5.	Research Institutions and Organizations:
o	Websites of research institutions and organizations focused on specific rare diseases often provide information about ongoing research, including molecular pathways.
6.	Rare Disease Patient Advocacy Groups:
o	These groups often provide resources and information about specific rare diseases, including the latest research findings and molecular pathways.
7.	Consult with Experts:
o	Reach out to medical geneticists, molecular biologists, and other experts in the field who specialize in rare diseases. They can provide insights and references related to molecular pathways.
8.	Collaborative Platforms:
o	Some platforms facilitate collaboration among researchers and clinicians working on rare diseases, allowing them to share information and findings. Examples include RD-Connect and IRDiRC.
Remember that research in the field of rare diseases is continually evolving, so it's crucial to verify the credibility of the sources and ensure that the information you're accessing is current and peer-reviewed. Additionally, a multidisciplinary approach involving genetics, molecular biology, and clinical expertise will provide a more comprehensive understanding of the molecular pathways associated with rare diseases.

**Comment by Chris-Evelo** [comment]
I think for starters we could organize the disease communities a bit in a hierarchical way. Like:

- Have disease on top 
- Have rare diseases below that 
- Have the inherited metabolic diseases below that again

And have each level have a clear pointer to the next lower (and maybe also next) higher level. 

Also in the case of rare diseases we could add a pointer to the material we have to use these pathways in pipelines and to the relevant tool sets.

When that is done the main thing will be to make sure we describe all that regularly in online texts and publications. That must be how ChatGPT knows about Reactome and KEGG as well. WikiPathways is the only one that actually has a specific rare disease collection and that should be mentioned more often.

**Comment by egonw** [comment]
SEO changes frequently, and with the new website Google was indexing the new pages only very slowly. This has been discussed multiple times in the WikiPathways developers meeting.

What generally still works: link a lot to the pathways on places indexed by Google. Google will see the links, and it indexed links that are linked to a lot. This was central to their original "page rank" concept, but guesstimate this still applies.

Also, we are planning to write a MCP next year, which will further support findability with LLMs.
